nym-api/src/v2/mod.rs

```diff
@@ -243,8 +215,7 @@ pub(crate) async fn start_nym_api_tasks_v2(config: &Config) -> anyhow::Result<Sh
         node_info_cache,
     });
 
-    // setup shutdowns
-    let (shutdown, axum_receiver) = ShutdownHandles::new(TaskManager::new(10));
+    let task_manager = TaskManager::new(10);
```

Comment:
suggestion: add a variable to give meaning to the `10`

Reply:
done